lib/internal/vm/source_text_module.js

```diff
@@ -37,29 +35,33 @@ const {
 } = binding;
 
 const STATUS_MAP = {
-  [kUninstantiated]: 'uninstantiated',
-  [kInstantiating]: 'instantiating',
-  [kInstantiated]: 'instantiated',
+  [kUninstantiated]: 'unlinked',
+  [kInstantiating]: 'linking',
+  [kInstantiated]: 'linked',
```

Comment:
nit: rename the constants?

Reply:
constant names are from v8, they're remapped there for the public api, which doesn't 100% line up with v8.